gobblin-utility/src/main/java/gobblin/util/ParallelRunner.java

```diff
@@ -246,7 +246,7 @@ public void close() throws IOException {
     } catch (InterruptedException ie) {
       throw new IOException(ie);
     } catch (ExecutionException ee) {
-      throw new IOException(ee);
+      throw new IOException(ee.getCause());
```

Comment:
Why make this change? I believe this won't preserve the stack trace of the original exception.